dart/pom.xml

```diff
@@ -35,6 +35,18 @@
       <artifactId>compile-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.parceler</groupId>
+      <artifactId>parceler-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+
+    <dependency>
+        <groupId>com.squareup</groupId>
+        <artifactId>javapoet</artifactId>
```

Comment:
Hmm, are there any class required at runtime in Dart?  Could you just make all of Dart `provided`?